RELEASE.md

```diff
@@ -0,0 +1,32 @@
+# Release process
```

Comment:
Perhaps it might make sense to put this together with the other internal contributions documentation? (https://dredd.org/en/latest/internals.html#contributing / https://github.com/apiaryio/dredd/blob/master/docs/internals.rst)

Reply:
Would it be relevant? That documentation addressed both internal and external contributors. The latter won't be able to publish to NPM to finish the release process. We'd get more reliability in the process if only the internal contributors release Dredd (less changes of changes to master while amidst the release).